aggregator/pom.xml

```diff
@@ -55,6 +55,18 @@
             <version>${project.version}</version>
             <classifier>${spark.version.classifier}</classifier>
         </dependency>
+        <!-- set provided scope for JNI and ucx to avoid pulling into aggregator jar  -->
```

Comment:
Alternatively we could simply add these artifacts to the exclude list of the shade artifactSet
```
                    <artifactSet>
                        <excludes>
                            <exclude>org.slf4j:*</exclude>
                            <exclude>com.nvidia:spark-rapids-jni:*</exclude>
                            <exclude>org.openucx:jucx:*</exclude>
                        </excludes>
                    </artifactSet>
```

Reply:
Yes, this one is more natural. Done.